docs/interpreter/jdbc.md

```diff
@@ -118,6 +118,11 @@ The JDBC interpreter properties are defined by default like below.
     <td>gpadmin</td>
     <td>The JDBC user name</td>
   </tr>
+  <tr>
+    <td>default.precode</td>
+    <td></td>
+    <td>Some SQL which executes while opening connection</td>
+  </tr>
```

Comment:
could you add an example of using this?
perhaps in ` ## Binding JDBC interpter to notebook`? Or another place you find